### PR TITLE
Restore dsp_offset and remove unused qurt code. Fixes #11134

### LINKED ISSUE #11134: Snapdragon timestamp offset
**Describe the bug**
Seems that timestamp synchronization between qurt and linux parts is broken.

**To Reproduce**
1. Start px4 on snapdragon
2. listener sensor_accel
3. observe that timestamp is `(18446744073701.398438 seconds ago)` taking in to account integer wrapping that is approximately 8 seconds in future

**Expected behavior**
Timestamps from uorb messages can be used directly without knowing if it was produced on qurt or Linux side.

**Log Files and Screenshots**
```
pxh> listener sensor_accel

TOPIC: sensor_accel instance 0 #1
 sensor_accel_s
        timestamp: 18235529088  (18446744073701.398438 seconds ago)
        error_count: 0
        device_id: 4288784 (Type: 0x41, UNKNOWN:2 (0x71)) 
        x: -0.2183
        y: 0.0912
        z: -9.6670
        integral_dt: 3997
        x_integral: -0.0007
        y_integral: 0.0004
        z_integral: -0.0385
        temperature: 0.0000
        scaling: -1.0000
        x_raw: -57
        y_raw: 330
        z_raw: -9753

```

**Drone (please complete the following information):**
- snapdragon eagle

**Additional context**
I also had a test subscribed to sensor_combined topic and immediately printed message timestamp and current `hrt_absolute_time()`. Previously the difference was ~400us now it is 8s. 

I found that there is  `dsp_offset` which was previously used but currently is unused.

### PR DESCRIPTION
**Pull request by karliss**
**Test data / coverage**
```
pxh> listener sensor_accel

TOPIC: sensor_accel instance 0 #1
 sensor_accel_s
        timestamp: 31655459637  (0.002233 seconds ago)
        error_count: 0
        device_id: 4288784 (Type: 0x41, UNKNOWN:2 (0x71)) 
        x: -0.2806
        y: -0.2188
        z: -9.7185
        integral_dt: 3
```

**Describe problem solved by the proposed pull request**
Fixes #11134 

**Describe your preferred solution**
I restored usage of dsp_offset, I guess that it might have been removed by accident during recent time/lockstep refactoring.

To reduce confusion also remove qurt drv_hrt.c as it hasn't been used more than a year.

### CODE REVIEW

File: platforms/posix/src/px4_layer/drv_hrt.cpp [hunk 1]
```diff
@@ -162,7 +162,11 @@ hrt_abstime _hrt_absolute_time_internal()
...
 	struct timespec ts;
 	px4_clock_gettime(CLOCK_MONOTONIC, &ts);
+#ifdef __PX4_QURT
+	return ts_to_abstime(&ts) + dsp_offset;
+#else
 	return ts_to_abstime(&ts);
+#endif
 }
 
 #ifdef __PX4_QURT
```
Comment (dagar) [inline platforms/posix/src/px4_layer/drv_hrt.cpp]: Super nitpicky, can you fix this whitespace while you're at it?

Comment (karliss) [reply platforms/posix/src/px4_layer/drv_hrt.cpp]: No problem. Did I forgot to setup git hooks again, or am I thinking about different project ?

Comment (dagar) [reply platforms/posix/src/px4_layer/drv_hrt.cpp]: Seems to slip through astyle, or at least the way we have it configured.

**Comment by julianoes** [comment]
Thanks @karliss, sorry I broke that.

**Pull request merged**